### PR TITLE
Fix/490 add IBM plex mono font face

### PR DESCRIPTION
**Pull request by craigyu**
# Description
Fixes: #490

A not so elegant solution to remove the font errors

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-spar-492-backend.apps.silver.devops.gov.bc.ca/)
[Frontend](https://nr-spar-492-frontend.apps.silver.devops.gov.bc.ca/)
[Oracle-API](https://nr-spar-492-oracle-api.apps.silver.devops.gov.bc.ca/)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "@babel/core": "^7.20.12",
     "@babel/plugin-syntax-flow": "^7.18.6",
     "@babel/plugin-transform-react-jsx": "^7.21.0",
-    "@bcgov-nr/nr-fsa-theme": "^1.1.2",
+    "@bcgov-nr/nr-fsa-theme": "^1.2.0",
```
Comment (mgaseta) [inline frontend/package.json:9]: Why downgrading the theme version here?

File: frontend/src/styles/custom.scss [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+  @include theme.theme();
+
+  @font-face {
+    font-display: "swap";
+    font-family: "IBM Plex Mono";
+    font-style: normal;
+    font-weight: 300;
+    src: url("https://1.www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Mono/fonts/split/woff2/IBMPlexMono-Light-Cyrillic.woff2") format("woff2"), url("https://1.www.s81c.com/common/carbon/plex/fonts/IBM-Plex-Mono/fonts/split/woff/IBMPlexMono-Light-Cyrillic.woff") format("woff");
+    unicode-range: "U+0400-045F", "U+0472-0473", "U+0490-049D", "U+04A0-04A5", "U+04AA-04AB", "U+04AE-04B3", "U+04B6-04BB", "U+04C0-04C2", "U+04CF-04D9", "U+04DC-04DF", "U+04E2-04E9", "U+04EE-04F5", "U+04F8-04F9";
+  }
```
Comment (mgaseta) [inline frontend/src/styles/custom.scss:50]: Can't we find font files for these fonts? At least `@font-face` won't be so huge...

**Review by mgaseta — CHANGES_REQUESTED** [review]
Agree that it's not as fancy as having the solution on the theme package... But at least solves our issues, good job!
Added only a few questions and a suggestion to make it less verbose.

File: frontend/package.json (hunk 1 above)
Comment (mgaseta) [reply frontend/package.json:9]: Oh, my bad hahaha. Nice then.

File: frontend/src/styles/custom.scss (hunk 2 above)
Comment (craigyu) [reply frontend/src/styles/custom.scss:50]: these are from @carbon's css file, using their resources to serve fonts is probably better

Comment (craigyu) [reply frontend/src/styles/custom.scss:50]: let me try if importing node module's css works

Comment (craigyu) [reply frontend/src/styles/custom.scss:50]: nope, doesn't work ^
I think this the "best" quick solution here, ideally something should be done on the theme package side

Comment (mgaseta) [reply frontend/src/styles/custom.scss:50]: Yeah, I agree, thanks for putting in the effort to try to reduce the code.

**Review by mgaseta — APPROVED** [review]
LGTM!

**Pull request merged**